modules/apps/frontend-css/frontend-css-custom-properties-sample-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -0,0 +1,152 @@
+// CCP Missings
```

Comment:
What's a CCP Missing?

Reply:
Classes that could go in [ccp-layer](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss)

Reply:
I have mixed feelings about this... because it means that the sample will react in a way that won't the rest of the system. This will likely result in failed expectations.

It'd be better to at least have these classes in `classic` for consistency, then, or properly use bootstrap's if they have them.

I'd say we can move these classes to the theme already so they can be used. Which also makes me think that the widget should probably live under `frontend-theme` and not `frontend-css`. 

Could you move it there? That way, we could even tweak the build process a little so we reuse the token lang keys we already generated.

Reply:
> Could you move it there? That way, we could even tweak the build process a little so we reuse the token lang keys we already generated.
No problem! I'm not sure how to config the build-process (I'll leave it to you 😄)

Reply:
Yeah, just move it and I'll take care of the final touches! 👍

Reply:
> I'd say we can move these classes to the theme already so they can be used. Which also makes me think that the widget should probably live under frontend-theme and not frontend-css.

Since I'll have to make important changes in the ccp-layer, do think I could postpone the classes relocation after the merging?